README.md

```diff
@@ -117,6 +117,15 @@ dev_setup.sh
 dev_setup.bat
```

Comment:
if we do add the `.env.sample`, it might be nice to include a step in the setup scripts that creates all of the `.env.*` for the developer.